packages/contracts/deploy/30_plugins/00_plugin-setups/11_token_voting_setup_conclude.ts

```diff
@@ -0,0 +1,39 @@
+import {ethers} from 'ethers';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {TokenVotingSetup__factory} from '../../../typechain';
+import {EHRE} from '../../../utils/types';
+
+const func: DeployFunction = async function (hre: EHRE) {
+  console.log(`\nConcluding token voting setup deployment.`);
+  const [deployer] = await hre.ethers.getSigners();
+
+  const {deployments} = hre;
+
+  const TokenVotingSetupDeployment = await deployments.get('TokenVotingSetup');
+  const tokenVotingSetup = await TokenVotingSetup__factory.connect(
+    TokenVotingSetupDeployment.address,
+    deployer
+  );
+
+  hre.aragonToVerifyContracts.push(TokenVotingSetupDeployment);
+  hre.aragonToVerifyContracts.push({
+    address: await tokenVotingSetup.implementation(),
+    args: [],
+  });
+  hre.aragonToVerifyContracts.push({
+    address: await tokenVotingSetup.governanceERC20Base(),
+    args: [
+      ethers.constants.AddressZero,
+      '',
+      '',
+      [ethers.constants.AddressZero, ethers.constants.AddressZero],
```

Comment:
Not sure if this will be correct. in the contrct, we do:

```
GovernanceERC20.MintSettings(new address[](0), new uint256[](0))
```

So they're empty arrays.

Reply:
you are right. done